src/common/util.ts

```diff
@@ -83,12 +83,16 @@ export async function findWorkDirectory(
 ): Promise<string> {
   const dirname = path.dirname(filePath)
   let patterns = [].concat(rootPatterns)
+  let dirCandidate = ''
   for (const pattern of patterns) {
     const [err, dir] =  await pcb(findup)(dirname, pattern)
-    if (!err && dir && dir !== '/') {
-      return dir
+    if (!err && dir && dir !== '/' && dir.length > dirCandidate.length) {
```

Comment:
It seems it's worth to filter the home directory too.

Reply:
If we will move this filter into `findWorkDirectory` then we probably change the behavior (tests are needed!). Now if for given directory its "working dir" is home dir, then it is skipped. After adding the comparison its "working dir" is it itself. 

In my opinion name "working dir" is confusing and should be changed to "project root" for function, constant and variable names. What do you think about this?

Reply:
Yes, we need test, but i do not have much time to do it.

> In my opinion name "working dir" is confusing and should be changed to "project root" for function, constant and variable names. What do you think about this?

make sense